scripts/vm/network/security_group.py

```diff
@@ -951,16 +951,15 @@ def parse_network_rules(rules):
   if rules is None or len(rules) == 0:
     return ret
 
-  lines = rules.split(';')[:-1]
+  lines = rules.split('NEXT;')[:-1]
```

Comment:
@wido what is the `NEXT` here? Can you give a sample of the rule/commands processed by this code?

Reply:
<pre>I:icmp;-1;-1;0.0.0.0/0,NEXT;I:tcp;0;65535;0.0.0.0/0,NEXT;I:udp;0;65535;0.0.0.0/0,NEXT;</pre>

Sure, this is how it looks like. The NEXT word is the last word before a new rule/line starts

Reply:
I think this extremely ugly solution is acceptable but i see a possible problem with the preceeding , it spells ',NEXT;' could it be ',NEXT,', instead? and if so, why not just ','? (read at your own risk)

Reply:
I think having the `,` is fine, @DaanHoogland see line 967/968, the cidrs could be comma separated, so splitting on `NEXT;` looks okay. The `NEXT` separator is added in `SecurityGroupRulesCmd.java`